hardhat.config.js

```diff
@@ -86,27 +86,54 @@ for (const f of fs.readdirSync(path.join(__dirname, 'hardhat'))) {
 const withOptimizations =
   argv.enableGasReport || argv.compileMode === 'production';
 
+
+if (Config.useFaucet){
+  console.log(`Use faucet: ${Config.faucetUrl}`)
```

Comment:
If you want to collect evaluation of faucet parameters in one place, then add checking that the faucet URL is not empty
like
```
if use faucet and faucet url is not empty...

else

```